docs/query-builder.txt

```diff
@@ -9,588 +9,1095 @@ Query Builder
    :values: tutorial
 
 .. meta::
-   :keywords: php framework, odm, code example
+   :keywords: code example, aggregation
 
-The database driver plugs right into the original query builder.
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
 
-When using MongoDB connections, you will be able to build fluent queries to
-perform database operations.
+Overview
+--------
 
-For your convenience, there is a ``collection`` alias for ``table`` and
-other MongoDB specific operators/operations.
+In this guide, you can learn how to use the {+odm-short+} extension of
+the Laravel query builder to work with a MongoDB database. The query builder
+lets you write queries for any supported database by using the same fluent
+interface and syntax.
```

Comment:
I think I had "a common" previously to represent the "fluent interface and syntax" which might be more clear than "the same". Let me know if one is more clear than the other, or how you might otherwise word it.

Reply:
maybe you can just add "... the same ... as the Laravel query builder"?

Reply:
Thanks for that perspective!

I wanted to describe the query builder rather than comparing it with something else, but it seems like using noun phrases (e.g. https://github.com/mongodb/laravel-mongodb/pull/2790#discussion_r1538001671) can cause some confusion.
In this sentence, the items being compared are syntax you use to access each of the supported databases rather than something with the query builder (given it's describing the query builder).

I'll spend a few minutes to try to reorder the sentence in a way that avoids connecting noun phrases to restrictive clauses.